catalog-rest-service/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -81,6 +86,11 @@
           "title": "DBT Manifest HTTP File Path",
           "description": "DBT manifest http file path to extract dbt models and associate with tables.",
           "type": "string"
+        },
+        "dbtRunResultsHttpPath": {
```

Comment:
Now that we added this params, should we update the descriptions of for example `dbtS3Config` to specify which new filename we will look for?

Reply:
Should we specify that in the docs instead of here ?

Reply:
let's do both. The descriptions appear on the UI form as well AFAIK

Reply:
Okay 👍